plenario_response/meta.py

```diff
@@ -0,0 +1,28 @@
+class Meta:
+
+    def __init__(self, json_payload: dict):
+        self.payload: dict = json_payload
+        self.params: dict = self.payload.get('params')
+        self.links: dict = self.payload.get('links')
+        self.counts: dict = self.payload.get('counts')
+
+    def get_page(self) -> int:
```

Comment:
I still think all of these should be `@property`. Using that doesn't expose or protect the value of the attribute beyond what anyone could do with it at any time. Wrapping a function with property just gives it cleaner syntax.

```python
class Meta:
    @property
    def page(self) -> int:
        return self.params.get('page')
```

If a user tried to set that value, a la `my_meta.page = 12`, it would blow up because there isn't a direct attribute on the object to be set.

Now if we _wanted_ to allow that, we would need a _setter_:

```python
@property
def page(self) -> int:
    return self.params.get('page')

@page.setter
def page(self, value: int) -> None:
    self.params['page'] = value
```